alembic

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+import sys
+from alembic.config import Config, CommandLine
+from webapp.database import DB_URL
```

Comment:
Hmm not sure this is worth it, slightly weird to import this here. the `DB_URL` constant looks so similar to the `DB_URL` environment variable that they could easily be confused. You might as well just do `os.environ.get("DB_URL")` in both places.

i think I prefer the `get_db_url` function you had before.